books/RayTracingInOneWeekend.html

```diff
@@ -2542,9 +2542,9 @@
 
 
 <div class='together'>
-If we take the gamut now we get a much more consistent ramp from darkness to lightness:
+Using this gamma correction, we now get a much more consistent ramp from darkness to lightness:
 
-  ![<span class='num'>Image 11:</span> The gamut of our renderer so far
+  ![<span class='num'>Image 11:</span> The gamma-corrected of two diffuse spheres
```

Comment:
"gamma-corrected <render/image> of two diffuse spheres"

or just

"The gamma-corrected rendering/result"

Reply:
I went with render just before you sent out this comment. ha